pkg/reconciler/pipelinerun/resources/apply.go

```diff
@@ -136,6 +146,17 @@ func ApplyReplacements(p *v1beta1.PipelineSpec, replacements map[string]string,
 	return p
 }
 
+func ApplyPipelineTaskReplacement(pt *v1beta1.PipelineTask, replacements map[string]string, arrayReplacements map[string][]string) *v1beta1.PipelineTask {
+	pt = pt.DeepCopy()
+	pt.Params = replaceParamValues(pt.Params, replacements, arrayReplacements)
+	for j := range pt.Conditions {
```

Comment:
Do you have use-cases for `retry-count` and `retries` in when expressions and conditions?  I am trying to think of some but am not coming up with anything right now.

If we don't have a strong idea of how they would be used for these fields I suggest only supporting them in `params` for now.

Reply:
Sorry, I can't either. Got it. I'll only support the replacement in `params`.